CHANGELOG.md

```diff
@@ -26,6 +26,12 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 # 3.0.3
 
+## API changes
+
+#### Search
+
+* The seqan3::shape is declared experimental, which means it's interface might change in future releases on SeqAn3. 
+
```

Comment:
It was never stable, so we do not need a changelog entry IMO

Reply:
Ok, I wasn't sure because the definition of done in the backlog is always the same and adding a changelog entry is always listed there.